### PR TITLE
Added support for laravel 5.6

### PR DESCRIPTION
**Pull request by duxthefux**

### CODE REVIEW

**Comment by duxthefux** [comment]
hey @ytake,

laravel 5.6 requires at least PHP 7.1.3, so I would recommend the test for PHP 7.0.

What do you think?

**Comment by ytake** [comment]
thanks!
I want to make it the same version (laravel) `"php": "^7.1.3"`

**Comment by ytake** [comment]
need return types and `declare(strict_types=1);` of ^PHP7.0(strict)

**Comment by duxthefux** [comment]
done :)

File: src/CreateFluentLogger.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+ * Created by PhpStorm.
+ * User: christopherfuchs
+ * Date: 14.02.18
+ * Time: 15:45
+ */
+
+namespace Ytake\LaravelFluent;
+
+use Fluent\Logger\FluentLogger;
+use Illuminate\Foundation\Application;
```
Comment (ytake) [inline src/CreateFluentLogger.php:13]: Do not use framework as a dependency
Please use container interface

**Comment by ytake** [comment]
#25 
There is an easier way

**Comment by duxthefux** [comment]
@ytake great. looks cleaner!. Gonna close mine.

**Pull request closed without merging**